dev-tools/compose/docker-compose.pedalboard.dev.yml

```diff
@@ -68,3 +68,12 @@ services:
       dockerfile: ${PROJECT_ROOT}/packages/discovery-provider/plugins/pedalboard/docker/Dockerfile.dev
     volumes:
       - ${PROJECT_ROOT}:/app
+
+  verified-notifications:
+    extends:
+      file: docker-compose.pedalboard.prod.yml
+      service: verified-notifications
+    build:
+      dockerfile: ${PROJECT_ROOT}/packages/discovery-provider/plugins/pedalboard/docker/Dockerfile.dev
+    volumes:
+      - ${PROJECT_ROOT}:/app
```

Comment:
There might be an issue with the `profile` field missing, we ran into this with ddex on newer versions of docker compose.

Try updating your local version of docker and running `audius-compose test run <any test suite>` - if it works just fine then don't worry about it.